src/main/java/com/epam/prd/jjop/helpers/objects/StreamOfIntegersCreator.java

```diff
@@ -0,0 +1,43 @@
+package com.epam.prd.jjop.helpers.objects;
+
+import java.util.Arrays;
+import java.util.stream.Stream;
+
+/**
+ * Given is array of String elements as a source for creating streams with {@code Arrays.stream(T[] array)} and
+ * {@code Stream.of(T... values)}.
```

Comment:
```suggestion
 * {@lin Stream#of(T... )}.
```

This syntax allows to jum between classes with CTRL+B.

Reply:
Here and anwards.